packages/edit-site/src/components/page-templates-template-parts/index.js

```diff
@@ -92,19 +98,24 @@ function normalizeSearchInput( input = '' ) {
 	return removeAccents( input.trim().toLowerCase() );
 }
 
-function TemplateTitle( { item, viewType } ) {
+function Title( { item, viewType } ) {
 	if ( viewType === LAYOUT_LIST ) {
 		return decodeEntities( item.title?.rendered ) || __( '(no title)' );
 	}
-
+	const linkProps = {
+		params: {
+			postId: item.id,
+			postType: item.type,
+			canvas: 'edit',
+		},
+	};
+	if ( item.type === TEMPLATE_PART_POST_TYPE ) {
+		linkProps.state = {
```

Comment:
I haven't seen this in use, what is it for? I presume it's for setting the path in case the users click the browser's back button after they've gone to the editor by clicking the title. Is this correct? If so, why templates don't need this?